internal/cli/service.go

```diff
@@ -454,6 +454,11 @@ func cmdService(
 			_ = pprof.Lookup("goroutine").WriteTo(os.Stderr, 1)
 			os.Exit(1)
 		}
+
+		if cerr := os.Stdin.Close(); cerr != nil {
+			logger.Errorf("failed to close stdin: %s", cerr)
+		}
```

Comment:
I figured it's a good idea to close stdin when shutting down...

Reply:
Nah lets not bother, I'm honestly not sure of any reason it should be done so happy to remove it entirely.